crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -367,13 +378,16 @@ impl<'a> Visitor<'a> {
         };
 
         let mut logger = self.sink.logger(behavior);
-        if self.run_opts.is_github_actions {
-            let package = if self.run_opts.single_package {
-                None
+        if let Some(vendor_behavior) = vendor_behavior {
+            let group_name = if self.run_opts.single_package {
+                ""
             } else {
-                Some(task_id.package())
+                task_id.package()
             };
-            let (header, footer) = github_header_footer(package, task_id.task());
+            let (header, footer) = (
+                (vendor_behavior.group_prefix)(group_name, task_start_time),
+                (vendor_behavior.group_suffix)(group_name, task_start_time),
```

Comment:
We'll want to defer computing the footer until later as we won't know until the task finishes what the correct task end time will be.

Given that this constraint complicates the design a bit, I think it's reasonable to either:
 - Leave this as-is with a TODO comment explaining that the timestamp will be incorrect
 - Remove the Gitlab CI vendor prefix/suffix implementation as it is the only one that uses the timestamp.

Reply:
I decided to remove the GitlabCI vendor for now. I'll be interested in adding it again in a separate pull request